docs/orchestration/global-hooks/guides.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Global Hooks Guide
+sidebarTitle: "Guide"
+---
+<img
+  alt="Penguin Fishing"
+  src="https://github.com/mage-ai/assets/blob/main/penguin_fishing.gif?raw=True"
+/>
+Follow this step by step guide on creating a global hook in Mage.
+
+### <b>Step 1: Navigate to the Global Hooks UI</b>
+1. Log in to your Mage account.
+1. From the left-hand sidebar, click on "Global Hooks" under the "Settings" section.
+1. Click on the "+ Add new global hook" button.
+
+### <b>Step 2: Configure the Hook</b>
+1. **Hook UUID**: Enter a unique name for your hook.
+2. **Resource Type**: Select the type of resource (API call type) from the resource drop down.
+3. **Operation Type**: Select the type of operation needed for your hook.
+4. **Targeting Conditions**: Enter conditions if you want it to apply the hook to only certain pipelines. Leave this field blank to apply the hook to all pipelines. 
```

Comment:
There's an additional step before getting to this part of configuring the hook. User needs to click "Create new global hook" button before the rest of the settings appear. We should  clarify that so users aren't confused where the rest of the settings are.
<img width="1103" alt="image" src="https://github.com/mage-ai/mage-ai/assets/78053898/b72e4188-4697-4c14-b498-8d11812846cd">

Reply:
Added a new note for this.
<img width="840" alt="Screenshot 2024-06-19 at 12 11 09 PM" src="https://github.com/mage-ai/mage-ai/assets/92221031/9ce7b70e-db66-4a00-8bd3-35465798a507">